src/scripts/components/explorer/explorer-store.jsx

```diff
@@ -3,6 +3,7 @@ import assign from 'object-assign';
 import AppDispatcher from '../../dispatcher/app-dispatcher';
 
 let audioList = [];
+let isAppending = false;
```

Comment:
I'm a little confuse. Why do we keep an isAppending state and have getTransitionState()? Not sure what their purpose is. Is there a scenario that would need this?